heliopy/data/mms.py

```diff
@@ -73,7 +75,12 @@ def fpi_dis_moms(probe, mode, starttime, endtime):
                             probe, mode, date.year, date.month, date.day, h)
             fnames.append(filename)
             dirs.append(this_relative_dir)
-
+            filename = ('mms{}_fpi_{}_l2_des-moms_'
```

Comment:
I'm confused by this addition, isn't `des-moms` a different data product to `dis-moms`?

Reply:
Yes - it's the (~thermal) electron moments, uploaded a goofed file apparently - fpi_dis_moms and fpi_des_moms could be different routines, or fpi_moms could fetch both.

Reply:
I think everything has been addressed, and it's done throwing errors over whitespace.  I'll keep poking at this and checking on CDAS version freshness, but I'm going to assume this is resolved.